stress/shell/src/main/java/alluxio/stress/cli/suite/MasterMaxThroughput.java

```diff
@@ -77,7 +77,7 @@ public void prepare() throws Exception {
     }
     try (JobMasterClient client = JobMasterClient.Factory.create(
         JobMasterClientContext.newBuilder(ClientContext.create(new InstancedConfiguration(
-            ConfigurationUtils.defaults()))).build())) {
+            ConfigurationUtils.copyDefaults()))).build())) {
```

Comment:
why not defaults?

Reply:
These are just from renaming `defaults` to `copyDefaults` I haven't checked through all instances to sort out whether a static copy is good enough vs it requires a copy yet. More cleanup will come later.